src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -87,40 +111,57 @@ public Drivetrain() {
     m_rotLimiter = new SlewRateLimiter(OIConstants.kRotationalSlewRate);
 
     m_timer.start();
-    m_prevTime = m_timer.get();
+    m_prevSlewRateTime = m_timer.get();
 
     m_odometry =
         new SwerveDriveOdometry(
             DriveConstants.kDriveKinematics,
             Rotation2d.fromRadians(-getGyroAngle().in(Units.Radians)),
-            new SwerveModulePosition[] {
-              m_frontLeft.getPosition(),
-              m_frontRight.getPosition(),
-              m_rearLeft.getPosition(),
-              m_rearRight.getPosition(),
-            });
+            m_swerveModulePositions,
+            m_pose);
+
+    configureAutoBuilder();
 
     m_powerDistribution.clearStickyFaults();
     SmartDashboard.putNumber("driveVelocity", 0);
   }
 
+  /** configures the pathplanner AutoBuilder */
+  private void configureAutoBuilder() {
+    AutoBuilder.configureHolonomic(
```

Comment:
Has this been tested and verified?

Reply:
it works

Reply:
That doesn't answer my question 🙂

Reply:
The autos haven't been run, but it doesn't crash the deployment